Libraries/LibWeb/CSS/CSSStyleValue.cpp

```diff
@@ -457,7 +457,21 @@ int CSSStyleValue::to_font_slope() const
             return oblique_slope;
         case Keyword::Normal:
         default:
-            break;
+            static int normal_slope = Gfx::name_to_slope("Normal"sv);
+            return normal_slope;
+        }
+    } else if (is_font_style()) {
+        switch (as_font_style().font_style()) {
+        case FontStyle::Italic:
+            static int italic_slope = Gfx::name_to_slope("Italic"sv);
```

Comment:
wow, this pattern is jank. that's a lot of static ints. A proper refactor to a better set of helper methods seems out of scope of the bugfix though.

Reply:
Agreed. In this PR though, we can get rid of the keyword-handling branch. [`Parser::parse_font_style_value()`](https://github.com/LadybirdBrowser/ladybird/blob/e2fa8cf7a862dc615cfa6c23716ed83ab1f8598e/Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp#L2613) only produces `FontStyleStyleValue`s now.

Once that's done, this is good to go! If you're interested in working on this further, that'd be good, but let's get this simple fix in first. :^)

Reply:
It's jank, yeah. I also just noticed that `Normal` isn't even an allowed `font_slope_name`, and `Oblique` doesn't use the actual slope specified in the `oblique` style...? Maybe a small refactor would fit in here.